copy-alerts/copy-alerts.sh

```diff
@@ -0,0 +1,73 @@
+#!/bin/bash
+set -euxo pipefail
+
+# Prompt user for namespace and validate
+read -p "Enter the namespace for your kubecost deployment: " NAMESPACE
+
+if [ -z "$NAMESPACE" ]; then
+    echo "Namespace cannot be empty. Exiting."
+    exit 1
+fi
+
+# Check if the provided namespace exists
+if ! kubectl get namespace "$NAMESPACE" &>/dev/null; then
+    echo "Namespace '$NAMESPACE' does not exist. Exiting."
+    exit 1
+fi
+
+OLD_POD=''
+NEW_POD=''
+if [[ $(kubectl -n "$NAMESPACE" get sts -l app=aggregator 2>&1) == *"No resources found"* ]]; then
+    echo "No aggregator StatefulSet found with label 'app=aggregator' in namespace $NAMESPACE"
+    # check if its single cluster deployment
+    if [ $(kubectl -n "$NAMESPACE" get service -l app=aggregator -o name | wc -l) -gt 0 ]; then
```

Comment:
is checking the labels on the service reliable?

Reply:
I created a new file called alerts-test-copy.json in aggregator nightly and then updated script to take that as the OLDFILENAME and NEWFILENAME with same suffix and it copied over. 

![Screenshot 2024-08-29 at 10 59 10 AM](https://github.com/user-attachments/assets/7bdb4afd-90ca-4ced-b44a-61c95f2cafc4)

![Screenshot 2024-08-29 at 11 00 11 AM](https://github.com/user-attachments/assets/1790e463-898a-426a-b345-a687966cedae)

![Screenshot 2024-08-29 at 11 00 06 AM](https://github.com/user-attachments/assets/d8305e92-6770-4c6a-8607-ed13baa0d28c)

I think our helm adds label app=aggregator [code-ref](https://github.com/kubecost/cost-analyzer-helm-chart/blob/develop/cost-analyzer/templates/aggregator-statefulset.yaml#L51) am i correct here ? 

It's also similar in script in [copy-reports.sh](https://github.com/kubecost/kubecost-utilities/blob/main/copy-reports/copy-reports.sh) . LMK if its enough evidence to answer your question if it's reliable.